models/art.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const artSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    limit: 200
+  },
+  startYear: {
+    type: String,
+  },
+  endYear: {
+    type: String,
+  },
+  ancestor: {
+    type: String
```

Comment:
`_ancestor` should be a reference to another Movement. Look back to the notes in `express-api` to see how to set up a relationship in Mongoose.